website_and_docs/content/documentation/selenium_manager.en.md

```diff
@@ -50,7 +60,9 @@ But there is even more. In addition to fixed browser versions (e.g., `113`, `114
 When these labels are specified, Selenium Manager first checks if a given browser is already installed (`beta`, `dev`, etc.), and when it is not detected, the browser is automatically managed.
 
 ## Configuration
-Like any other CLI tool, Selenium Manager is invoked from the shell, and different arguments can be used to specify specific capabilities. You can check the different arguments supported by Selenium Manager by running the following command:
+**TL;DR**: Selenium Manager should work silently and transparently for most users. Nevertheless, there are scenarios (e.g., when a proxy or firewall forbids making network requests) where custom configuration is required.
```

Comment:
The bindings do support Proxies out of the box if users specify the proxy information directly in the Options class (as opposed to using args or environment variables)

Reply:
Changed that TLDR.